# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1456337371 18000
#      Wed Feb 24 13:09:31 2016 -0500
# Node ID 9f838471a5da23d17cb210aea5a0e8b894b2e03f
# Parent  6228796397bca29ddbea518da72c1a809e5f7e45
* kpse.cc: Avoid warning about static function in "header" file.

diff --git a/liboctave/util/kpse.cc b/liboctave/util/kpse.cc
--- a/liboctave/util/kpse.cc
+++ b/liboctave/util/kpse.cc
@@ -2181,17 +2181,17 @@ dir_links (const std::string& fn)
 #endif
     }
 
   return ret;
 }
 
 #endif /* WIN32 */
 
-static void
+static inline void
 do_subdir (str_llist_type *str_list_ptr, const std::string& elt,
            unsigned elt_length, const std::string& post)
 {
 #ifdef WIN32
   WIN32_FIND_DATA find_file_data;
   HANDLE hnd;
   int proceed;
 #else
